### PR TITLE
chore(create-medusa-app, product, utils, medusa-cli): Resolve `string-width` dep issue

### PR DESCRIPTION
**Pull request by shahednasser**

### CODE REVIEW

File: packages/product/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
     "jest": "^25.5.4",
     "medusa-test-utils": "^1.1.40",
     "pg-god": "^1.0.12",
-    "rimraf": "^5.0.0",
+    "rimraf": "^3.0.2",
```
Comment (olivermrbl) [inline packages/product/package.json:45]: @adrien2p we had to downgrade rimraf in the product module package due to some issues with clashing dependencies. Do you foresee any issues with that?

Comment (olivermrbl) [reply packages/product/package.json:45]: cc @carlos-r-l-rodrigues @riqwan

**Review by riqwan — APPROVED** [review]
works as expected locally for me

**Review by olivermrbl — APPROVED** [review]

File: packages/product/package.json (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply packages/product/package.json:45]: It is used only to delete the folder `/dist` before building. All good.

**Review by carlos-r-l-rodrigues — APPROVED** [review]

**Pull request merged**